docs/tech-manual/content/configuration.tex

```diff
@@ -220,18 +220,19 @@ \section{Configuration}
 		%
 		\endlastfoot
 		%
-		Storage service port & 8063 & LP\_ETL\_STORAGE\_PORT \\
-		Storage domain & http://localhost:8060 & LP\_ETL\_DOMAIN \\
+		Storage service port & 8063 & LP\_ETL\_STORAGE\_PORT \\		
 		Executor service port & 8065 & LP\_ETL\_EXECUTOR\_PORT \\
-		Executor domain & http://localhost:8060 & LP\_ETL\_DOMAIN \\
-		Executor monitor service port & 8061 & LP\_ETL\_MONITOR\_PORT \\
-		Executor monitor domain & http://localhost:8060 & LP\_ETL\_DOMAIN \\
+		Executor monitor service port & 8061 & LP\_ETL\_MONITOR\_PORT \\		
 		Service UI port & 8060 & LP\_ETL\_PORT \\
-		UI domain & http://localhost:8060 & LP\_ETL\_DOMAIN \\* \bottomrule
+		Service domain & http://localhost:8060 & LP\_ETL\_DOMAIN \\* \bottomrule
 		\caption{LinkedPipes ETL services configurations}
 		\label{tab:my-table5}\\
 	\end{longtable}
 	
+	Note that it is important to change the LP\_ETL\_DOMAIN each time the deployment environment changes. If it runs locally please set the value to be \textit{http://localhost:8060}, otherwise if it runs on a dedicated domain, the variable value must be set accordingly. If this is not done, the services will run but will not be able to load any pipelines because of a mismatch between the domain in the pipeline URI (created by using the variable value) and the domain of the service host. 
```

Comment:
you can use `\texttt{LP\_ETL\_DOMAIN}` to use monospace text and signify that this is a env variable.

Reply:
Italics have been used so far for this purpose